data/json/effects.json

```diff
@@ -1919,5 +1919,10 @@
         "base_mods": {
             "speed_mod": [50]
         }
+    },
+    {
+        "type": "effect_type",
+        "id": "has_prospectus",
+        "desc": ""
```

Comment:
Shouldn't it be `[""]`?

Reply:
"desc" shouldn't be in the definition at all, since "show_in_info" is false. Updated, thanks.